global/navbarElements.php

```diff
@@ -54,49 +57,56 @@
                         <a class = "transparent"><i class="fa fa-cubes fa-fw"></i><b> Convenor </b></a>  
                     </li>
                     <ul class="nav nav-second-level">';
+    $convStudents = '   <li>

```

Comment:
Holy shit we should've split these up into components, screw constantly throwing HTML into php variables like this, it's barely readable.

Reply:
ahahaha at least at the time we knew this was festy

Reply:
Yeah lmao, the concept is sound but not putting all the variables/markup etc into 1 file. In a way we were kinda doing react style shit here, throw a bunch of markup into variables etc.